package/src/components/CartEmptyMessage/v1/CartEmptyMessage.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { applyTheme } from "../../../utils";
+
+const EmptyButton = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const EmptyMessage = styled.div`
+  color: ${applyTheme("color_coolGrey500")};
+  display: flex;
+  font-size: ${applyTheme("cartEmptyMessageFontSize")};
+  justify-content: center;
+  letter-spacing: ${applyTheme("cartEmptyMessageLetterSpacing")};
+  margin-bottom: ${applyTheme("cartEmptyMessageMarginBottom")};
+`;
+
+class CartEmptyMessage extends Component {
+  static propTypes = {
+    /**
+     * On object of component children to pass into this component
+     */
+    components: PropTypes.shape({
+      ContinueShoppingButton: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
+    }),
+    /**
+     * Onclick function to pass to the Button component. Not handled internally, directly passed
+     */
+    onClick: PropTypes.func
+  }
+
+  static defaultProps = {
+    onClick() {}
+  };
+
+  render() {
+    const { onClick } = this.props;
+    const { ContinueShoppingButton } = this.props.components;
+
+    return (
+      <Fragment>
+        <EmptyMessage>Your shopping cart is empty.</EmptyMessage>
```

Comment:
User's might want to provide a custom message, make this a prop and default it to "Your shopping cart is empty."